tox.ini

```
@@ -2,14 +2,14 @@
 envlist=
   linting
   py{35,36,37,38,39}-pytestlatest
-  py38-pytestmaster
+  py38-pytestmain
   py38-psutil
 
 [testenv]
 extras = testing
 deps =
   pytestlatest: pytest
-  pytestmaster: git+https://github.com/pytest-dev/pytest.git@main
+  pytestmain: git+https://github.com/pytest-dev/pytest.git@main
```

@nicoddemus doesn't this work without a branch name?

```suggestion
  pytestmain: git+https://github.com/pytest-dev/pytest.git
```

---

Probably yes, this might be a left over when switched from having a `features` branch.